OpenSim/Simulation/Manager/Manager.h

```diff
@@ -95,38 +95,29 @@ class OSIMSIMULATION_API Manager
     double _ti;
     /** Final time of the simulation. */
     double _tf;
-    /** First dt in an integration. */
-    double _firstDT;
     
     /** Storage for the states. */
     std::unique_ptr<Storage> _stateStore;
 
-   int _steps;
-   /** Number of integration step tries. */
-   int _trys;
-   /** Maximum number of steps in an integration. */
-   int _maxSteps;
-   /** Flag for signaling a desired halt. */
-   bool _halt;
-   /** Minimum step size. */
-   double _dtMin;
```

Comment:
Noted. Oddly enough, `ForwardTool` does respect `_maxDT`, so I think this is just an oversight.

This might bring up a separate question then: should `AnalyzeTool` derive from `AbstractTool`? `AnalyzeTool` also won't do anything with `_minDT` or `_maxDT`.